test/acceptance/test/install_tests.go

```diff
@@ -81,8 +81,16 @@ Global Flags:
 	It("Verify that gitops quits if flux-system namespace is present", func() {
 		var errOutput string
 		namespace := "flux-system"
+		tip := generateTestInputs()
 
 		defer deleteNamespace(namespace)
+		defer deleteRepo(tip.appRepoName, gitProvider, gitOrg)
+
+		By("And application repo does not already exist", func() {
```

Comment:
Question - it's not clear to me what we are testing here.  deleteRepo() is a test util function?

Reply:
@palemtnrider  This is a pattern that has been used in many other acceptance tests. I was trying to be consistent. My guess is that we need to make sure the repo we are going to be testing with does not exist already. Probably this was a problem in the early stages where tests were not so stable. Nowadays it seems unlikely that we would have an existing repo with the random name used.